### DISCUSSION TITLE
Make Neutralino Standalone on Linux

### DISCUSSION THREAD

**Discussion by iacore** [neutralinojs/neutralinojs#747] [Q&A] [answered]
How does `neu` invoke `neutralino-linux_x64`? This is related to #745 .

**Comment by shalithasuranga** [comment, marked as answer]
Hey.. I guess you meant about `neu run`. The neu CLI just sets some CLI params and spawn a child process at https://github.com/neutralinojs/neutralinojs-cli/blob/70edd77198c9b57c804fdc244ca5042800969129/src/modules/runner.js#L43
You can see all supported CLI args at https://neutralino.js.org/docs/cli/internal-cli-arguments
